README.md

```diff
@@ -313,18 +374,18 @@ var pjax = new Pjax({
   switchesOptions: {
     ".js-Pjax": {
       classNames: {
-        // class added on the element that will be removed
+        // class to be removed from the element during replacement
         remove: "Animated Animated--reverse Animate--fast Animate--noDelay",
-        // class added on the element that will be added
+        // class to be added on the element during replacement
```

Comment:
This change makes it less clear. These classes are added to the new element that is replacing the old one, e.g. a fade in.

Reply:
Same as above -